src/backend/distributed/sql/udfs/citus_unmark_object_distributed/latest.sql

```diff
@@ -1,6 +1,10 @@
-CREATE FUNCTION pg_catalog.citus_unmark_object_distributed(classid oid, objid oid, objsubid int)
+CREATE FUNCTION pg_catalog.citus_unmark_object_distributed(classid oid, objid oid, objsubid int,checkobjectexistence boolean)
```

Comment:
We should probably do `checkobjectexistence boolean DEFAULT true` so that rolling upgrades don't break. I can explain what I mean in chat when you're available.